MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,781 @@
+
+You can change the color for the watermark in the editor by using the `PlaceholderColor` property in `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Description")
+            {
+                e.DataFormItem.PlaceholderText = "Enter description";
+                e.DataFormItem.PlaceholderColor = Colors.MistyRose;
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Padding
+
+You can create a space around a label and editor by using `Padding` property in the `DataFormItem`.
+
+Here, `FirstName` data field will be changed from the default position.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.Padding = new Thickness(10);
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing label name
+
+By default property name will be displayed as label text. You can change the label text by using display attribute or using the `GenerateDataFormItem` event.
+
+#### Using attribute
+
+You can change the label text by setting the `Name` or `ShortName` in display attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    
+    [Display(Name = "FirstName")]
+    public string Name { get; set; }
+    
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+You can change the label text by using the `LabelText` property in `DataFormItem` .
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "Name")
+            {
+                e.DataFormItem.LabelText = "FirstName";
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Remove Label for an editor
+
+By default, the label will be generated by the property name. You can remove label text by using the DataFormDisplayAttribute or using the `GenerateDataFormItem` event.
+
+#### Using attribute
+
+You can remove the label text for editor by setting `ShowLabel` as false in `DataformDisplayOption` attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string MiddleName { get; set; }
+    
+}
+{% endhighlight %}
+{% endtabs %}
+
+#### Using events
+
+You can remove the label text for editor by using the `ShowLabel` property in `DataFormItem`.
```

Comment:
You can hide the label text for editor by using the `ShowLabel` property in `DataFormItem`

Reply:
content updated